### ISSUE TITLE
Good bug in your implementation of AdamP

### ISSUE THREAD

**Issue by jstjohn** [clovaai/AdamP#14]
I noticed that you never use the `step_size` (https://github.com/clovaai/AdamP/blob/master/adamp/adamp.py#L81) which makes use of the `bias_correction1` on `beta1`, which inflates the learning rate in the initial steps of training (see https://arxiv.org/abs/2110.10828). Instead you leave the learning rate as is, only applying `bias_correction2` which inflates the estimated variance early in training, which lowers the learning rate in the first few epochs. I don't think you should fix this bug, but it may be helpful to just add a comment or something saying that you skip this step, and maybe even throw a link to my paper stating that this is not a bug :)

**Comment by jstjohn** [comment]
Never mind, I found where it comes in: https://github.com/clovaai/AdamP/blob/master/adamp/adamp.py#L98
